airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -56,6 +57,23 @@
 USE_PER_RUN_LOG_ID = hasattr(DagRun, "get_log_template")
 
 
+# These are the only keys that are allowed in the elasticsearch_configs section
+# This should be synchronized with config section in provider.yaml
+VALID_ES_CONFIG_KEYS = {
+    "http_compress",
+    "verify_certs",
+}
```

Comment:
I've just worry that this allow-list could also breaks users pipelines.

For example @cesar-vermeulen 's config contains additional values

```ini
[elasticsearch_configs]
max_retries = 3
retry_timeout = True
timeout = 30
```

Reply:
BTW, I've found yesterday that we have [`[logging] remote_task_handler_kwargs` ](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#remote-task-handler-kwargs) in the first glance it is good candidate to replace entire `elasticsearch_configs` config section in case of Airflow 2.6+ so user could provide additional `kwargs` by this parameter.

Reply:
How about now @Taragolis ? I am now filtering configuration by the signature of the Elasticsearch constructor.